### ISSUE TITLE
excalidraw.app is unavailable, Google App not working

### ISSUE THREAD

**Issue by evil-pechenka** [excalidraw/excalidraw#7622]
Applications in Google Cloud are not opening due to the unavailability of excalidraw.app.

**Comment by evil-pechenka** [comment]
https://workspace.google.com/marketplace/app/excalidraw/1017678967926

Yes. I thought this was official integration.

**Comment by ad1992** [comment]
We aren't actively maintaining it and probably will be sunsetting it soon so we would suggest you to migrate your drawings to https://excalidraw.com/ or https://plus.excalidraw.com/

In the meantime, @dwelle can you look into this?

**Comment by evil-pechenka** [comment]
This advice seems useless to me, how can I transfer the drawings if I don't have access to them.

Last week I drew a flowchart, now it's inaccessible.

**Comment by ad1992** [comment]
> This advice seems useless to me, how can I transfer the drawings if I don't have access to them.
> 
> Last week I drew a flowchart, now it's inaccessible.

You can download those `.excalidraw` files in local and directly open them in  https://excalidraw.com/ or https://plus.excalidraw.com/ and it should work.

**Comment by evil-pechenka** [comment]
Understood. Thank you, have a good day. 🥰

**Comment by evil-pechenka** [comment]
> You can download those `.excalidraw` files in local and directly open them in https://excalidraw.com/ or https://plus.excalidraw.com/ and it should work.

I downloaded the file, but unfortunately https://excalidraw.com/ does not open it, saying that the file format is incorrect. I found another document — it also does not open. It seems my flowchart is lost.

**Comment by ad1992** [comment]
> > You can download those `.excalidraw` files in local and directly open them in https://excalidraw.com/ or https://plus.excalidraw.com/ and it should work.
> 
> I downloaded the file, but unfortunately https://excalidraw.com/ does not open it, saying that the file format is incorrect. I found another document — it also does not open. It seems my flowchart is lost.

can you share the file (if its not confidential)?

**Comment by alcedo** [comment]
im facing the same issue as well. exported .excalidraw file cant be opened by excalidraw.com

**Comment by ad1992** [comment]
> Hello! Here you have a corrupt file, can you have a look at it please? Thank you! [BackOffice BaaS.zip](https://github.com/excalidraw/excalidraw/files/14100890/BackOffice.BaaS.zip)

@oscarsubirats thanks for sharing the file, I have fixed it and you can check it [here](https://excalidraw.com/#json=_e0lyUBnvV12tboPJzgug,-Z_wiQ9B4mFtEp-_nTQknQ), you just need to add the `type: excalidraw` in the json and it should work.

Let me know if the data retrived is correct.

@alcedo @evil-pechenka can you also make sure if `type: excalidraw` is present in the file ?

**Comment by evil-pechenka** [comment]
> @alcedo @evil-pechenka can you also make sure if `type: excalidraw` is present in the file ?

Greetings! I am not a programmer (surprisingly, non-programmers do exist), I have never worked with JSON files in my life, and I cannot insert anything into them. In order for me to insert something, I need step-by-step instructions: what to open it with, and where to insert it.

Thanks to the Almighty, ChatGPT appeared — I simply gave it the file, and it presented it in the form of readable lists. I copied them into another application. Therefore, I do not need help.

Yes, it is very unfortunate that you are discontinuing the application without warning, but we managed!

Praise ChatGPT! (Soon, it will replace programmers.)

**Comment by alcedo** [comment]
@evil-pechenka yes it works! thanks. i added it to the top of the file just before the elements array.

**Comment by tzographos** [comment]
Hello, how can I set my google drive account as the storage provider in https://app.excalidraw.com  ?
